src/internal/connector/sharepoint/data_collections.go

```diff
@@ -104,22 +105,27 @@ func collectLists(
 	if scope.Matches(selectors.SharePointSite, siteID) {
 		spcs := make([]data.Collection, 0)
 
-		dir, err := path.Builder{}.Append(siteID).
-			ToDataLayerSharePointPath(
-				tenantID,
-				siteID,
-				path.ListsCategory,
-				false)
+		tuples, err := preFetchListIDs(ctx, serv, siteID)
 		if err != nil {
-			return nil, errors.Wrapf(err, "failed to create collection path for site: %s", siteID)
+			return nil, err
 		}
+		// Prefetch
```

Comment:
Seems like the prefetch already happened?

Reply:
Resolved with [02b11fd](https://github.com/alcionai/corso/pull/1788/commits/02b11fddc486ca263a35bfdfeb2c879f4bd88875)